eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -204,6 +204,12 @@ stages:
       - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
       - download: current
         artifact: ${{parameters.ArtifactName}}-signed
+      - task: NuGetCommand@2
+        displayName: 'Publish to DevOps Feed'
+        inputs:
+          command: push
```

Comment:
Didn't you also want to add this step to the release step when we are publishing to nuget itself? I know that is a different issue but they seem related enough to go ahead and do it with this PR.

Reply:
Done